dowhy/gcm/stats.py

```diff
@@ -161,3 +164,65 @@ def permute_features(feature_samples: np.ndarray,
             np.random.shuffle(feature_samples[:, feature])
 
     return feature_samples
+
+
+def estimate_f_test_p_value(X_training_a: np.ndarray,
```

Comment:
I understand the motivation here, but you may consider `estimate_ftest_pvalue` for ease of use.